_studio/mfx_lib/vpp/include/mfx_vpp_hw.h

```diff
@@ -636,7 +633,7 @@ namespace MfxHwVideoProcessing
                 m_frcRational[VPP_IN]  = frcRational[VPP_IN];
                 m_frcRational[VPP_OUT] = frcRational[VPP_OUT];
 
-                m_minDeltaTime = MFX_MIN((__UINT64) (m_frcRational[VPP_IN].FrameRateExtD * MFX_TIME_STAMP_FREQUENCY) / (2 * m_frcRational[VPP_IN].FrameRateExtN),
+                m_minDeltaTime = std::min((__UINT64) (m_frcRational[VPP_IN].FrameRateExtD * MFX_TIME_STAMP_FREQUENCY) / (2 * m_frcRational[VPP_IN].FrameRateExtN),
```

Comment:
__UINT64? can't we use uint64_t here?

Reply:
I think we can, but it should be different PR

Reply:
Added clean up of these types in #1251